torch/_inductor/ir.py

```diff
@@ -3349,6 +3349,46 @@ def get_template_tiling(self):
         )
 
 
+class AllReduce(ExternKernelAlloc):
+    kernel = "torch.ops.c10d.traceable_allreduce"
+    #1 constant_args show up in codegen, use those if needed
+    #2 allreduce_ can't be called from python, without fixing bindings
+    #3 try implementing traceable_allreduce and adding POD types to api for rank/size?
+    #4 think about what to do for cpu
```

Comment:
is this still an issue if you're using c10d that deals with backend (nccl/gloo) selection?

Reply:
i think this concern goes away if the `dist` route works fully (and is efficient)